docker/benchmark-runner/startup-runner.sh

```diff
@@ -20,4 +20,5 @@ source $VIRTUALENV_DIR/bin/activate
 # The runner runs at a higher priority than other processes to ensure that it's
 # able to finish infrastructure tasks regardless of the fuzzing workload.
 export RUNNER_NICENESS="-5"
+apt-get update && apt-get install -y rsync
```

Comment:
This won't work as this script is run on a device without network access. Can you install this in `docker/base-image/Dockerfile` please? That way every image we use gets rsync